### PR TITLE
promote gcp-compute-persistent-disk-csi-driver v1.2.3

### PR DESCRIPTION
**Pull request by cpanato**
- v1.2.3: https://github.com/kubernetes-sigs/gcp-compute-persistent-disk-csi-driver/releases/tag/v1.2.3

/assign @mattcary @msau42 @spiffxp

### CODE REVIEW

File: k8s.gcr.io/images/k8s-staging-cloud-provider-gcp/images.yaml [hunk 1]
```diff
@@ -1,14 +1,16 @@
 - name: gcp-compute-persistent-disk-csi-driver
   dmap:
+    "sha256:02ffdc33eaa1c53342d9962ac216a3482bb42aaae12871371bc9594ce94577a2": ["v1.3.2-rc1"]
```
Comment (mattcary) [inline k8s.gcr.io/images/k8s-staging-cloud-provider-gcp/images.yaml:3]: Do we promote release candidates? I thought we just tested them out of staging, then promoted the actual release only.

Comment (cpanato) [reply k8s.gcr.io/images/k8s-staging-cloud-provider-gcp/images.yaml:3]: we can promote, I can drop that as well

Comment (mattcary) [reply k8s.gcr.io/images/k8s-staging-cloud-provider-gcp/images.yaml:3]: Yeah, let's just promote the final candiate and not the RCs to keep the image registry a little cleaner.

Comment (cpanato) [reply k8s.gcr.io/images/k8s-staging-cloud-provider-gcp/images.yaml:3]: sgmt!

Comment (cpanato) [reply k8s.gcr.io/images/k8s-staging-cloud-provider-gcp/images.yaml]: done @mattcary thanks for the review! PTAL

**Comment by mattcary** [comment]
/lgtm
/approve

Thanks!

(I'm not actually sure if I have approval power, we'll see...)

**Comment by cpanato** [comment]
you have :)

**Comment by mattcary** [comment]
Hey, cool! :-)

**Pull request merged**